### DISCUSSION TITLE
What are you think:  Check for unused #includes in axmol source to reduce compile and link times.

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#2202] [General]
What are you think:  Check for unused #includes in axmol source to reduce compile and link times.
Maybe the  `#include axmol.h` is nice but includes to much for own projects?

**Comment by smilediver** [comment]
I would get rid of `axmol.h` completely, but maybe that's too extreme. At least the Axmol itself should not use it.
